src/middleware/emailValidation.js

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi';
+
+export default {
+  /**
+     * @description validates request body before registration
+     * @param {object} req
+     * @param {object} res
+     * @param {object} next
+     * @returns {object} validation oject
+     */
+
+  emailValidation(req, res, next) {
```

Comment:
the validation rules for password, confirm password and email have been defined in validation.js. These should be defined separately and reused here and in the userSchema. This way we'll have the same rules being applied in both workflows

Reply:
Worked but not yet pushed